charts/akri/0.10.4/Chart.yaml

```diff
@@ -4,3 +4,4 @@ description: A Helm chart for Akri
 name: akri
 type: application
 version: 0.10.4
+icon: https://raw.githubusercontent.com/suse-edge/charts/_images/akrilogo.png
```

Comment:
This won't survive a `make charts`, and I think we should add the icon in the upstream chart (it's in my todo list to do a PR for this)

Reply:
I'll remove it then :) It will use the placeholder meanwhile